static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -0,0 +1,201 @@
+@import '../../icons/video-icons-svg';
+
+@mixin hide {
+    visibility: hidden;
+    opacity: 0;
+}
+
+@mixin show {
+    visibility: visible;
+    opacity: 1;
+}
+
+@mixin fade-in {
+    @include show;
+    transition: opacity 1s linear, visibility 0s;
+}
+
+@mixin fade-out {
+    @include hide;
+    transition: opacity 1s linear, visibility 1s;
+}
+
+.atom--media--youtube {
+	position: relative;
+
+	.atom--media--youtube--iframe,
+	.atom--media--youtube--overlay,
+	.atom--media--youtube--placeholder {
+        height: 100%;
+        width: 100%;
+        position: absolute;
+	}
+
+	.atom--media--youtube--iframe {
+		top: 1px;
+        z-index: 0;
+
+        &:not(.youtube__video-ready),
+        &.youtube__video-ended {
+            ~ .atom--media--youtube--overlay {
+                .atom--media--youtube--play-button,
+                .atom--media--youtube--bottom-bar {
+                     @include hide;
+                }
+            }
+        }
+
+        &.youtube__video-ready:not(.youtube__video-started) {
+            ~ .atom--media--youtube--overlay {
+                cursor: pointer;
+
+                .atom--media--youtube--play-button,
+                .atom--media--youtube--bottom-bar {
+                    @include fade-in;    
+                }
+            }
+        }
+
+        &.youtube__video-started:not(.youtube__video-ended) {
+            ~ .atom--media--youtube--overlay {
+                .atom--media--youtube--placeholder,
+                .atom--media--youtube--play-button,
+                .atom--media--youtube--bottom-bar {
+                    @include fade-out;
+                    transition-delay: 500ms;
+                    transition-duration: 500ms;     
+                }
+            }
+        }
+
+        &.youtube__video-ended.youtube__video-started {
+            ~ .atom--media--youtube--overlay {
+                .atom--media--youtube--play-button,
+                .atom--media--youtube--bottom-bar {
+                    @include fade-in;
+                }
+            }
+        }
+	}
+
+    .atom--media--youtube--overlay {
+        pointer-events: none;
+    }
+
+	.atom--media--youtube--placeholder {
+		background-size: cover;
+        background-position: 50% 50%;
+        background-repeat: no-repeat;
+        z-index: 1;
+        text-align: center;
+	}
+
+    .atom--media--youtube--play-button {
```

Comment:
Given that this is the same styling as our current play button, is it worth writing this twice? Given that it is likely we will keep the old styling, I would prefer that we have as little duplication as possible to still keep it readable.